pkg/metricutil/metricutil_test.go

```diff
@@ -45,7 +58,7 @@ func (s *testMetricsSuite) TestConvertName(c *C) {
 	}
 
 	for _, input := range inputs {
-		c.Assert(input.newName, Equals, convertName(input.name))
+		c.Assert(input.newName, Equals, camelCaseToSnakeCase(input.name))
```

Comment:
Change it to `c.Assert(camelCaseToSnakeCase(input.name), Equals, input.newName)`. Obtained value should be in front of expected value. 
Although it is not introduced by you, thanks for helping to fix it.